content/source/docs/cloud/run/modes-and-options.html.md

```diff
@@ -19,12 +19,13 @@ Terraform Cloud runs support many of the same modes and options available in the
 
 > **Hands-on:** Try the [Use Refresh-Only Mode to Sync Terraform State](https://learn.hashicorp.com/tutorials/terraform/refresh) tutorial on HashiCorp Learn.
 
--> **Version note:** Refresh-only support is available in Terraform Enterprise v202106-1 or later and requires a workspace using at least Terraform CLI v0.15.2.
+-> **Version note:** Refresh-only support is available in Terraform Enterprise v202106-1 or later and requires a workspace using at least Terraform CLI v0.15.4.
```

Comment:
Oh whoops, good catch - it was [`replace` that landed in 0.15.2](https://github.com/hashicorp/terraform/blob/v0.15/CHANGELOG.md#0152-may-05-2021), not `refresh-only`!

Reply:
Hmm, maybe I should update that below, while I'm in here!